diff --git a/dist/debug.js b/dist/debug.js
index 31f4e4bf1b81f0914d29e8ef839eab9dfd9d3fb9..cfd2ee4bee15367573694af95f6554033a029862 100644
--- a/dist/debug.js
+++ b/dist/debug.js
@@ -3,6 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.Debug = void 0;
 const createDebug = require("debug");
 /**
- * Debug instance with the given namespace: http-proxy-middleware
+ * Debug instance with the given namespace: http-proxy-middleware.
+ * Remove this patch when this is ok - https://github.com/nuxt/nuxt/issues/15608
  */
-exports.Debug = createDebug('http-proxy-middleware');
+exports.Debug = Object.assign(createDebug('http-proxy-middleware'), {extend: ()=>{return ()=> {}}});